lumium-space/src/components/menu/TopDrawerMenu.tsx

```diff
@@ -1,26 +1,98 @@
-import { Drawer, DrawerOverlay, DrawerContent, DrawerHeader, DrawerBody, useDisclosure, IconButton } from '@chakra-ui/react';
-import { ArrowDownIcon } from '@chakra-ui/icons';
+import {
+    Drawer,
+    DrawerBody,
+    DrawerFooter,
+    DrawerHeader,
+    DrawerOverlay,
+    DrawerContent,
+    DrawerCloseButton,
+    useDisclosure,
+    Input,
+    IconButton,
+    Flex,
+    Spacer,
+    Menu,
+    MenuButton,
+    MenuList,
+    MenuItem,
+    MenuItemOption,
+    MenuGroup,
+    MenuOptionGroup,
+    MenuDivider,
+    Button,
+    NumberInput,
+    NumberInputField,
+    NumberInputStepper,
+    NumberIncrementStepper,
+    NumberDecrementStepper,
+} from '@chakra-ui/react'
+import { ArrowRightIcon } from '@chakra-ui/icons';
+import { signOut } from "supertokens-auth-react/recipe/thirdpartyemailpassword";
 import React from 'react';
 
+async function onLogout() {
+    await signOut();
+    window.location.href = "/";
```

Comment:
replace by 

```suggestion
Router.push('/');
```

have to `import Router from 'next/router';`